### PR TITLE
Create a container image and push it.

### PR DESCRIPTION
**Pull request by nohoster**
Hey there, I really like your project and wanted to help a little bit. I wanted to use it myself so I created a Dockerfile to build an image and then decided to help you.
The github action is already configured to your repository and triggers on new tags. You only need to enable write permissions on the repository settings/actions/general and at the bottom there is Workflow permissions to enable write permissions.

### CODE REVIEW

**Comment by nohoster** [comment]
Oh, I forgot to mention I have a container image ready at my repository in case you want to check it. It's multiarch, adm64 and arm64, the action uses a matrix to do parallel building to speed the process and the container image uses distroless for security and less size.

File: .dockerignore
Comment (cr4yfish) [inline .dockerignore:1]: Can you add the following ignores or give me write perms to the fork?
```
# ignore the android build folder
android

# ignore env files
.env.local
next-env.d.ts

# ignore service worker files
public/sw.*
```

**Review by cr4yfish — CHANGES_REQUESTED** [review]
Thank you very much for contributing! I will merge as soon as you add the missing ignores.
:)

**Comment by nohoster** [comment]
I've added the ignores.

**Comment by cr4yfish** [comment]
Thank you very much for your contribution @nohoster !

**Review by cr4yfish — COMMENTED** [review]
Why is the action not running?

**Comment by cr4yfish** [comment]
Nvm,  missed this somehow lol
```
  tags:
    - 0.*
```

**Pull request merged**